docs/source/API/core/KokkosConcepts.rst

```diff
@@ -102,9 +102,7 @@ and ``OpenMPTarget``, the current state of the Kokkos |ExecutionSpaceTwo|_ conce
 
     template <typename Ex>
     concept ExecutionSpace =
-        CopyConstructible<Ex> &&
-        DefaultConstructible<Ex> &&
-        Destructible<Ex> &&
+        std::regular<Ex> &&
```

Comment:
not consistent with
>     * execution spaces are semi-regular now
but this is just in the pr description

Reply:
fixed